src/Message/Part.php

```diff
@@ -179,14 +179,14 @@ public function getDecodedContent($keepUnseen = false)
 
             // If this part is a text part, try to convert its encoding to UTF-8.
             // We don't want to convert an attachment's encoding.
-            if ($this->getType() === self::TYPE_TEXT
-                && strtolower($this->getCharset()) != 'utf-8'
-            ) {
-                $this->decodedContent = Transcoder::create()->transcode(
-                    $this->decodedContent,
-                    $this->getCharset()
-                );
-            }
+            // if ($this->getType() === self::TYPE_TEXT
+                // && strtolower($this->getCharset()) != 'utf-8'
+            // ) {
+                // $this->decodedContent = Transcoder::create()->transcode(
+                    // $this->decodedContent,
+                    // $this->getCharset()
+                // );
+            // }
```

Comment:
After commenting this code no test fails, and [testEncoding7Bit](https://github.com/ddeboer/imap/blob/master/tests/MessageTest.php#L41) passes: @ddeboer do you have any test case that justify this piece of code?

Reply:
This was a fix for #3, so also `testSubjectEncoding()`.

Reply:
I don't know what changed since november 2013, but the is that if you removing code doesn't break tests, it's either:

1. the tests are not enough
1. the code is unnecessary

But don't be afraid: we need a lot of test cases with different charset to test against, and I will gather and test them, in future PR, of couse before the any next release.

Just not now: before deeping into the charset topic, we need minor improvement like coding standards and re-enable code-coverage; one step at time

Reply:
But if the tests pass without this, let’s remove it.

Reply:
Right: I hate commented code too and for sure will be removed soon.
One step at time, reminder added: https://github.com/ddeboer/imap/issues/174